app/bundles/CoreBundle/Command/PullTransifexCommand.php

```diff
@@ -107,7 +107,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
                             $translationsConnector = $transifex->get('translations');
                             $response              = $translationsConnector->getTranslation('mautic', $alias, $language);
                             $translation           = json_decode((string) $response->getBody(), true);
-                            $path                  = $translationDir.$language.'/'.$bundle.'/'.basename($file);
+                            $filename              = str_replace('.ini', '.'.$language.'.ini', basename($file));
+                            $path                  = $translationDir.$language.'/'.$bundle.'/'.$filename;
```

Comment:
The translations files have names <domain>.<language>.<file extension>, e.g. message.de.ini. Files with these names can be loaded into the translation catalog without hard coding in the [translation service](https://github.com/volha-pivavarchyk/mautic/blob/fix/fallback-locale-issue/app/bundles/CoreBundle/Translation/Translator.php#L47).

I changed the description. I tryed to explain how to test it. And I can write tests for it

Reply:
Tests would be really good idea. I know it will conflict with https://github.com/mautic/mautic/pull/11447 in M5 so if you want to ensure this will keep working in M5 we need the test.

Also, the product team has decided that all PRs including bug fixes must be open against the 5.x branch. Only those you want to back-port to M4 should have duplicated PR against the 4.4 branch.